specifications/api/swagger.yaml

```diff
@@ -6096,6 +6045,24 @@ paths:
       responses: *responsesGetShelleyKey
 
   /wallets/{walletId}/policy-key:
+    post:
+      operationId: postPolicyKey
+      tags: ["Transactions"]
+      summary: Create
+      description: |
+        <p align="right">status: <strong>stable</strong></p>
+
+        Create policy key for the wallet. Used for public account key derived
+        wallets or already instantiated mnemonic wallets to set policy public key
```

Comment:
```suggestion
        Create policy key for the wallet. 
        
        In order to be able to mint/burn assets with `POST Construct` endpoint there needs to be 
        a policy key set for the wallet. Invoking this endpoint would be required for all wallets instantiated
        before introducing mint/burn feature prior to making a mint/burn transaction from them.

```

Reply:
now, for the consistency I added optional parameter like in getPolicyKey.
I would advocate for maintaining `getPolicyKey` in line with `postPolicyKey`. Could be helpful for consumers of the API, also POST writes to DB, and GET is reading

Reply:
Perhaps `POST` should be actually `PUT`... it doesn't create new one every time, but sets one if doesn't exists (or overwrites the old one, but it is always the same if I'm not mistaken)